### PR TITLE
[BUGFIX] - remove object and kickstarter

### PR DESCRIPTION
**Pull request by michaelBenin**
Hi,

We needed this update as we found fitvid to be causing issues with other embeds. 

Thank you!

### CODE REVIEW

**Comment by michaelBenin** [comment]
See how fitvids wrapped this embed:

http://www.fastcompany.com/3046400/how-to-be-a-success-at-everything/the-top-25-us-cities-for-jobs-this-year

(note this will be fixed monday morning from this fork)

**Comment by rosszurowski** [comment]
Hey. Thanks for contributing.

I see the issue at the link you shared. It looks like Tableau swaps out an `object` tag for an `iframe`. When the page loads fitvids wraps the `object`, but Tableau then removes it and replaces it with an unwrapped `iframe`.

Since this is a really specific case, I'm not inclined to change the default selectors to solve it. Instead, I'll add an `ignore` option so you can solve your problem like so:

```javascript
fitvids({
  ignore: ['object']
})
```

File: index.js [hunk 1]
```diff
@@ -4,9 +4,7 @@
 var selectors = [
 	'iframe[src*="player.vimeo.com"]',
 	'iframe[src*="youtube.com"]',
-	'iframe[src*="youtube-nocookie.com"]',
-	'iframe[src*="kickstarter.com"][src*="video.html"]',
```
Comment (rosszurowski) [inline index.js]: I don't follow why the Kickstarter selector needs to be removed. I put together a [test case](http://jsbin.com/laxepudefa/edit?html,js,output) that only removes the `object` line, and it seems to work as expected. Is there something I'm missing?

**Comment by michaelBenin** [comment]
Ignore meets my needs. Regarding kickstarter, I didn't see the need for it.

**Pull request closed without merging**